scimono-server/src/main/java/com/sap/scimono/entity/Resource.java

```diff
@@ -62,15 +62,17 @@
   private static final Predicate<Map.Entry<String, Extension>> CUSTOM_EXTENSION_FILTER = extension -> extension.getKey()
       .startsWith(Schema.EXTENSION_SCHEMA_URN);
 
+  private final String resourceType;
```

Comment:
There is already existing resource type attribute defined by SCIM protocol. It is part of the **meta** sub-attributes. More info [here](https://tools.ietf.org/html/rfc7643#section-3.1)

Can you use it instead?

Reply:
We could use the resourceType attribute from the meta object. But the meta are computed after validation phase by the API. I'd rather leave it as it is.
What do you think ?

Reply:
It will be better if you just reorder the execution `ResourceCustomAttributesValidator<User>.validate(...)` and the place where meta was set. See:
- `Users.createUser(...)` 
- `Users.updateUser(...)`
- `Groups.createGroup(...)` 
- `Groups.updateGroup(...)`

Reply:
Ok I have updated the PR to retrieve the resource type from the meta data.